clients/cobol-lsp-vscode-extension/src/commands/ClearCopybookCacheCommand.ts

```diff
@@ -16,27 +16,27 @@ import * as path from "node:path";
 import * as vscode from "vscode";
 import { FileType } from "vscode";
 import {
-  C4Z_FOLDER,
   CLEARING_COPYBOOK_CACHE,
   COPYBOOK_CACHE_CLEARED_INFO,
   COPYBOOKS_FOLDER,
+  ZOWE_FOLDER,
 } from "../constants";
+import { Utils } from "../services/util/Utils";
 
 /**
- * Clears the downloaded copybook cache folder ({workspace}/.c4z/.copybooks).
+ * Clears the downloaded copybook cache folder ({Utils.getExtensionFolder()}/.zowe/.copybooks).
  *
  */
 export function clearCache() {
   vscode.window.setStatusBarMessage(
     CLEARING_COPYBOOK_CACHE,
     Promise.resolve().then(
       () => {
-        const firstWorkspaceFolder = vscode.workspace.workspaceFolders![0];
-        const folderUri = firstWorkspaceFolder!.uri;
-        const fileUri = folderUri.with({
-          path: path.join(folderUri.fsPath, C4Z_FOLDER, COPYBOOKS_FOLDER),
+        const folderUri = vscode.Uri.file(Utils.getExtensionFolder());
+        const zoweFileUri = folderUri.with({
+          path: path.join(folderUri.fsPath, ZOWE_FOLDER, COPYBOOKS_FOLDER),
```

Comment:
onlyt the `.copybooks` directory?

Reply:
could you please expand this ?

Reply:
I am just wondering why delete only the `.copybooks` folder and not the `.zowe`.

Reply:
Because this function clears copybooks not all files related to zowe api. For future use I think it is better to keep as keep folder.